File: Turculescu_Samuel/Week_5/Domain/Entities/Employee.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Text;
using System.Text.Json.Serialization;

namespace Persistence.EfCore
{
    public partial class Employee
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public string Job { get; set; }
        public string EmployeeId { get; set; }
        public string Password { get; set; }
        public int RestaurantId{ get; set; }


        public virtual Restaurant Restaurant { get; set; }
    }
}

File: Turculescu_Samuel/Week_8/order-and-pay-client/src/models/employee.ts
import { Restaurant } from "./restaurant";

export type Employee = {
    id: string,
    firstName: string,
    lastName: string,
    email: string,
    phone: string,
    job: string,
    employeeId: string,
    password: string,
    restaurantId: string,

    restaurant: Restaurant
}